cmd_test.go

```diff
@@ -22,29 +26,51 @@ const (
 	cmdExampleArgs = "arg1 arg2"
 )
 
+func waitMessages(t *testing.T, count int, errorCount int) {
+	for {
+		select {
+		case reply := <-replies:
+			msgs = append(msgs, reply)
+		case err := <-cmdError:
+			errs = append(errs, err)
+		case <-time.After(1 * time.Second):
+			t.Fatal("Timed the shit out")
```

Comment:
Can this print out msgs and errors so far received?

Reply:
@toolateforteddy yep, I think that this is a good idea, also forgot to clean up this message. Too much 🍺